spark.yaml

```
@@ -48,7 +49,13 @@ pipeline:
     with:
       patches: make-distribution.patch bump-packages.patch
```

```suggestion
      patches: make-distribution.patch
```

Same as comment below, I don't think? the bump-packages.patch gets used anywhere now with pombump doing the driving

---

It does get used, there are some other kind of patching it does. For example:

```
+              <excludes>
+                <exclude>com.google.common.util.concurrent.internal.**</exclude>
+              </excludes>
```

---

We can add more capabilities into pombump as we go along.
But the above file also patches a `.scala` file.